### ISSUE TITLE
Client unable to send video to room

### ISSUE THREAD

**Issue by dereklucas** [twilio/twilio-video.js#1214]
- [x] I have verified that the issue occurs with the latest twilio-video.js release and is not marked as a known issue in the [CHANGELOG.md](https://github.com/twilio/twilio-video.js/blob/master/CHANGELOG.md).
 - [x] I reviewed the [Common Issues](https://github.com/twilio/twilio-video.js/blob/master/COMMON_ISSUES.md) and open GitHub issues and verified that this report represents a potentially new issue.
 - [x] I verified that the [Quickstart](https://github.com/twilio/video-quickstart-js) application works in my environment.
 - [x] I am not sharing any [Personally Identifiable Information (PII)](https://www.twilio.com/docs/glossary/what-is-personally-identifiable-information-pii)
       or sensitive account information (API keys, credentials, etc.) when reporting this issue.

**Expected behavior:**

- A client that can join a room should be able to send video to that room.

**Actual behavior:**

A client joined a room, but was unable to send video or audio tracks. They were listed as a participant for the other participants, but they were never seen. When they ran the Twilio Network Test most things said they passed, except for:
- Video: Test Group Room with TURN
- Video: Test Group Room without TURN

Both of these tests said "Successfully joined a Room" and 5 seconds later "BytesReceived (Audio): 0" and the same for video. Then there is an error saying "Group Room failed: No stats were able to be recorded".

Room SIDs:

Arcivr Production Rooms exhibiting this issue:
RMbaf0d5263922a7d84fd6af73c89a79ca
RM6b3b53929cbe0ba4bf4793d8d876a1ba

Twilio Network Test Room:
RMde1b50bfece85baa521183cfdeda2254

**Software versions:**

 - [x] Browser(s): Chrome, Desktop Safari, Mobile Safari
 - [x] Operating System: Mac OS 10.15.7, iOS 13
 - [x] twilio-video.js: 2.7.2
 - [x] Third-party libraries (e.g., Angular, React, etc.): React

**Comment by makarandp0** [comment]
Hello @dereklucas, Thank you for reporting this issue. 
Failures from network test point to connectivity issues. In this case looks like clients firewall is preventing connectivity with  the  media server. Is this issue limited to a specific customer / network ?  Can you  please have them verify that the addresses and ports [listed here](https://www.twilio.com/docs/video/ip-addresses) are not blocked by their firewall.  

I looked at the room logs for RM6b3b53929cbe0ba4bf4793d8d876a1ba, but only saw one participant publishing tracks - Can you specify the participant sid that is having issues sending/receiving tracks ? Any logs from the client console would be helpful too.

Thanks,
Makarand

**Comment by manjeshbhargav** [comment]
Closing due to no activity.

**Comment by dereklucas** [comment]
Apologies for the lack of follow up: we've confirmed that this was an issue with one of the clients connecting to the Twilio servers, so it is not a `twilio-video.js` issue.
